client/consensus/manual-seal/src/lib.rs

```diff
@@ -19,7 +19,13 @@
 //! A manual sealing engine: the engine listens for rpc calls to seal blocks and create forks.
 //! This is suitable for a testing environment.
 
+#![allow(dead_code)]
```

Comment:
What is this for?

Reply:
Because manual-seal is not used by default. If we don't have this line of code, compiler will warn us the code is there but not being used.

Reply:
FYI pubic items can be unused in a library.

Reply:
I removed this line. It was actually hiding a valuable warning that `rpc::ManualSeal` was no longer public which is also fixed.